### PR TITLE
Mohammadkalash dearborn

### PR DESCRIPTION
**Pull request by mohammadkalash**
@JasonMick What do you think of my list?  Do you have any other recommendations?

### CODE REVIEW

**Review by sparris9 — CHANGES_REQUESTED** [review]
Needs more Emojis

**Review by hectorsector — CHANGES_REQUESTED** [review]
Agreed, not enough 👍

File: mohammadkalash-Dearborn.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+###Where to Eat
+
+- Sponge Docks
+- Dunedin Brewery :Beers:
+- LA Bistro
```
Comment (devops23345) [inline mohammadkalash-Dearborn.md:6]: Is LA Bistro a fusion place?

**Review by sparris9 — APPROVED** [review]

**Comment by mohammadkalash** [comment]
@hectorsector Can you review my updates?

**Pull request merged**